drivers/storage/portworx/components_test.go

```diff
@@ -11647,8 +11647,6 @@ func TestPodDisruptionBudgetEnabled(t *testing.T) {
 		// Node that does not return a scheduler node name, does not get counted towards quorum calculation as we cannot
 		// determine if it is part of the current k8s cluster or not.
 		{NonQuorumMember: false, NodeLabels: map[string]string{pxutil.NodeLabelPortworxVersion: "3.1.0"}},
-		// Node that is not part of the cluster, does not get counted towards quorum calculation.
-		{NonQuorumMember: false, SchedulerNodeName: "node6", NodeLabels: map[string]string{pxutil.NodeLabelPortworxVersion: "3.1.0"}},
```

Comment:
We should actually keep the case and just update the comment above "Node that is not part of the k8s cluster, still gets counted towards quorum calculation".
We will have to change the required value below to 3 instead of 2.

This will test your changes in this PR.

Reply:
I have added it now